src/main/resources/buildServerResources/js/buildScan.js

```diff
@@ -0,0 +1,23 @@
+BS.BuildScan = {
+    startCleanup: function () {
+        if (!confirm("This operation may require significant time, and the server will be less performant during the clean-up.\n" +
+            "Are you sure you want to start clean-up process now?")) return false;
+        $('cleanupResult').style.visibility = 'visible'
+        $('startBuildScanCleanupButton').disabled = true
+        $('cleanupResult').textContent = 'Cleanup is running...';
+        $('progressRing').style.visibility = 'visible'
```

Comment:
Flip the two lines above (just cosmetics)

Reply:
done